<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>queryStu</class>
 <widget class="QDialog" name="queryStu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,6,2">
     <item>
      <widget class="QComboBox" name="comboBox_method">
       <item>
        <property name="text">
         <string>请选择查询方式</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>按姓名</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>按学号</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>按性别</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>按年龄</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>按院系</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_content"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_search">
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
